ext/storage_sources/s3_store/s3_log_system.cpp

```diff
@@ -33,55 +62,48 @@ S3LogSystem::Log(Aws::Utils::Logging::LogLevel logLevel, const char *tag, const
     va_end(args);
 }
 
+// Writes the log stream to the output stream.
 void
 S3LogSystem::LogStream(
   Aws::Utils::Logging::LogLevel logLevel, const char *tag, const Aws::OStringStream &messageStream)
 {
     LogAwsMessage(tag, messageStream.rdbuf()->str().c_str());
 }
 
+// Directs the output stream to WiredTiger's log streams.
 void
 S3LogSystem::LogAwsMessage(const char *tag, const std::string &message) const
 {
     _wtApi->err_printf(_wtApi, NULL, "%s : %s", tag, message.c_str());
 }
 
+// Directs the output stream to WiredTiger's log streams matched at WiredTiger's log stream levels.
 void
 S3LogSystem::LogVerboseMessage(int32_t verbosityLevel, const std::string &message) const
 {
     if (verbosityLevel <= _wtVerbosityLevel) {
-        /* Use err_printf for error and warning messages and use msg_printf for notice, info and
-         * debug messages. */
-        if (verbosityLevel < -1)
+        // Use err_printf for error and warning messages and use msg_printf for notice, info and
+        // debug messages.
+        if (verbosityLevel < WT_VERBOSE_NOTICE)
             _wtApi->err_printf(_wtApi, NULL, "%s", message.c_str());
         else
             _wtApi->msg_printf(_wtApi, NULL, "%s", message.c_str());
     }
 }
 
-void
-S3LogSystem::LogErrorMessage(const std::string &message) const
-{
-    LogVerboseMessage(WT_VERBOSE_ERROR, message);
-}
-
-void
-S3LogSystem::LogDebugMessage(const std::string &message) const
-{
-    LogVerboseMessage(WT_VERBOSE_DEBUG, message);
-}
-
+// Sets the WiredTiger verbosity level by mapping the AWS SDK log level.
 void
 S3LogSystem::SetWtVerbosityLevel(int32_t wtVerbosityLevel)
 {
     _wtVerbosityLevel = wtVerbosityLevel;
-    /* If the verbosity level is out of range it will default to AWS SDK Error level. */
+    // If the verbosity level is out of range it will default to AWS SDK Error level.
     if (verbosityMapping.find(_wtVerbosityLevel) != verbosityMapping.end())
         _awsLogLevel = verbosityMapping.at(_wtVerbosityLevel);
     else
         _awsLogLevel = Aws::Utils::Logging::LogLevel::Error;
 }
 
+// Unused.
```

Comment:
:) Can we add a note on why we need it if we don't use it. Maybe it is required to get the code compiled?